script.js

```diff
@@ -0,0 +1,21 @@
+const menuAction = document.querySelectorAll(".navbar-mobile li");
+let listMenu = Array.from(menuAction)
+
+function open() {
+    const mobileMenu = document.querySelector(".navbar-mobile");
+    mobileMenu.classList.remove("hidden");
+}
+
+const menuList = document.querySelector(".mobile-nav");
+menuList.addEventListener("click", open);
+
+const closeMenu = document.querySelector("#cross-sign");
+
+
+function close() {
+    const mobileNav = document.querySelector(".navbar-mobile");
+    mobileNav.classList.add("hidden")
+}
+closeMenu.addEventListener("click", close)
+
+listMenu.forEach((elem) => elem.addEventListener("click", close))
```

Comment:
## [OPTIONAL]
- [ ]  To keep your code clean I think it would be nice to put all variable declarations (`const varableName`) at the top of your file.